frappe/commands/site.py

```diff
@@ -19,24 +19,24 @@
 @click.option('--db-type', default='mariadb', type=click.Choice(['mariadb', 'postgres']), help='Optional "postgres" or "mariadb". Default is "mariadb"')
 @click.option('--db-host', help='Database Host')
 @click.option('--db-port', type=int, help='Database Port')
-@click.option('--mariadb-root-username', default='root', help='Root username for MariaDB')
-@click.option('--mariadb-root-password', help='Root password for MariaDB')
+@click.option('--db-root-username', default='root', help='Root username for MariaDB or PostgreSQL, Default is "root"')
+@click.option('--db-root-password', help='Root password for MariaDB or PostgreSQL')
```

Comment:
```suggestion
@click.option('--db-root-username', '--mariadb-root-username', default='root', help='Root username for MariaDB or PostgreSQL, Default is "root"')
@click.option('--db-root-password', '--mariadb-root-password', help='Root password for MariaDB or PostgreSQL')
```

We can do this without breaking the CLI ;)

---

Ref: 

<img width="943" alt="Screenshot 2022-02-15 at 11 46 01 AM" src="https://user-images.githubusercontent.com/36654812/154003482-9ac9d74e-e3aa-4cb6-ae37-ef11ba7903ad.png">